test/unit/models/test_superblocks.py

```diff
@@ -220,7 +220,8 @@ def test_deterministic_superblock_creation(go_list_proposals):
 
     max_budget = 60
     prop_list = Proposal.approved_and_ranked(proposal_quorum=1, next_superblock_max_budget=max_budget)
-    sb = dashlib.create_superblock(prop_list, 72000, budget_max=max_budget, sb_epoch_time=misc.now())
+    maxgovobjdatasize = dashd.govinfo['maxgovobjdatasize']
```

Comment:
The unit tests don't have access to DashCore JSONRPC, so you will have to mock the maxgovobjdatasize (hard-code the value in the tests)

Reply:
fixed